source/operations/install-deploy-manage/modify-minio-tenant.rst

```diff
@@ -10,87 +10,59 @@ Modify a MinIO Tenant
+      You can use ``helm list -n TENANT-NAMESPACE`` to validate the Tenant name.
 
-   To customize the TLS certificates mounted on the MinIO Tenant, enable the :guilabel:`Custom Certificates` switch.
-   Select the :guilabel:`Add Certificate +` button to add custom TLS certificates.
+      See :ref:`minio-tenant-chart-values` for more complete documentation on the available Chart fields.
 
-Add Trusted Certificate Authorities
-   The MinIO Tenant validates the TLS certificate presented by each connecting client against the host system's trusted root certificate store.
-   The MinIO Operator can attach additional third-party Certificate Authorities (CA) to the Tenant to allow validation of client TLS certificates signed by those CAs.
+   .. tab-item:: Operator Console
 
-   To customize the trusted CAs mounted to each Tenant MinIO pod, enable the :guilabel:`Custom Certificates` switch.
-   Select the :guilabel:`Add CA Certificate +` button to add third party CA certificates.
+      .. versionchanged:: 6.0.0
 
-   If the MinIO Tenant cannot match an incoming client's TLS certificate issuer against either the container OS's trust store *or* an explicitly attached CA, MinIO rejects the connection as invalid.
+         The MinIO Operator Console has been deprecated and removed.
 
+      To modify the Tenant, use ``kubectl apply -k`` or ``kubectl edit`` against the Tenant object in the namespace.
 
-Manage Tenant Pools
--------------------
+      The following command exports the Tenant YAML:
 
-Specify Runtime Class
-~~~~~~~~~~~~~~~~~~~~~
+      .. code-block:: shell
 
-.. versionadded:: Console 0.23.1
+         kubectl get tenant TENANT-NAME -n TENANT-NAMESPACE -o yaml > TENANT-NAME.yaml
 
-When adding a new pool or modifying an existing pool for a tenant, you can specify the :kube-docs:`Runtime Class Name <concepts/containers/runtime-class/>` for pools to use.
+      You can edit the YAML object using your preferred text editor and apply to to update the Tenant:
 
-.. Following link is intended for K8s only
-.. _minio-decommissioning:
+      .. code-block:: shell
+         
+         kubectl apply -f TENANT-NAME.yaml
 
-Decommission a Tenant Server Pool
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-MinIO Operator 4.4.13 and later support decommissioning a server pool in a Tenant.
-Specifically, you can follow the :minio-docs:`Decommission a Server pool <minio/linux/operations/install-deploy-manage/decommission-server-pool.html>` procedure to remove the pool from the tenant, then edit the tenant YAML to drop the pool from the StatefulSet.
-When removing the Tenant pool, ensure the ``spec.pools.[n].name`` fields have values for all remaining pools.
-
-.. include:: /includes/common-installation.rst
-   :start-after: start-pool-order-must-not-change
-   :end-before: end-pool-order-must-not-change
+      You can also use ``kubectl edit`` to 'live edit' Kubernetes objects.
```

Comment:
The single quote marks look odd. I'd swap them for " or just drop the emphasis altogether.